### ISSUE TITLE
Network parameter initialization

### ISSUE THREAD

**Issue by zhijiejia** [GeoZcx/A-deeply-supervised-image-fusion-network-for-change-detection-in-remote-sensing-images#9]
Hi @GeoZcx, I find a phenomenon that when I initialization of network parameters, the network can't detect any change in Pytorch Version.  But when I don't initialize of network parameters the network works well. The follow is  the code of initialization 

`   
 def _initialize_weights(self):

        for m in self.modules():
            if isinstance(m, nn.Conv2d) or isinstance(m, nn.ConvTranspose2d) or isinstance(m, nn.Linear):
                nn.init.kaiming_normal_(m.weight, mode='fan_out', nonlinearity='relu')
                if m.bias is not None:
                    m.bias.data.zero_()
            elif isinstance(m, nn.BatchNorm2d):
                m.weight.data.fill_(1)
                m.bias.data.zero_()
`

**Comment by zhijiejia** [comment]
Oh, I have a idea that may I make the pre training parameters of vgg16 are initialized again. .....

**Comment by zhijiejia** [comment]
Another problem I found is that when the above parameter initialization method is used, the network can't be non convergent. I don't know why。。

**Comment by zhijiejia** [comment]
It may be that dropout, p = 0.6, brings too much instability. p value should be reduced appropriately.

**Comment by GeoZcx** [comment]
We use the pre-trained VGG16 parameter as the deep feature extraction network parameter, only the feature fusion network needs initialization. If initialize the whole network as you do, the network would take a long time to converge.

**Comment by fpan98** [comment]
how to only initialize parameter except vgg16
